typescript/sdk/src/middleware/account/schemas.ts

```diff
@@ -10,3 +14,11 @@ export const AccountConfigSchema = z.object({
   routerOverride: ZHash.optional(),
   ismOverride: ZHash.optional(),
 });
+
+export const GetCallRemoteSettingsSchema = z.object({
```

Comment:
Maybe rename this? Having trouble understanding what this is for from the name. Or leave a comment

Reply:
Added comment `/* For InterchainAccount::getCallRemote() */`